float8_experimental/fsdp_utils.py

```diff
@@ -0,0 +1,61 @@
+import math
+import warnings
+from typing import List
+
+import torch
+import torch.nn as nn
+from float8_experimental.float8_dynamic_utils import WeightWithDynamicFloat8CastTensor
+from float8_experimental.float8_linear import Float8Linear
+from float8_experimental.float8_linear_utils import linear_requires_sync
+from float8_experimental.float8_utils import EPS
+
+
+def precompute_float8_scale_for_fsdp(module: nn.Module) -> None:
```

Comment:
Should we add a `@torch.no_grad()` decorator on this?

Reply:
good catch. adding @torch.no_grad()